### PR TITLE
ENDOC-25 ENG-561 Add the documents related to selecting the embedded …

### PR DESCRIPTION
**Pull request by nshaw**
…databases.

### CODE REVIEW

**Comment by nshaw** [comment]
@Kerruba , this is meant to combine the changes from PRs #92, #94, and #96 but moved into the new docs structure.

File: vuepress/docs/docs/getting-started/README.md [hunk 1]
```diff
@@ -164,6 +164,16 @@ Helm is a package manager for Kubernetes that helps you define, install, and upg
 curl -L -C - -O https://dev.entando.org/assets/yaml/entando.yaml
 ```
 
+### EntandoCompositeApp
+
+Entando uses a Kubernetes Custom Resource named `EntandoCompositeApp` that is the core of the Entando infrastructure. It's composed of 3 parts:
```
Comment (Kerruba) [inline vuepress/docs/docs/getting-started/README.md:169]:
```suggestion
To quickly deploy an application, Entando uses a Kubernetes Custom Resource named `EntandoCompositeApp`. It's composed of 3 parts:
```
Originally the EntandoCompositeApp was meant to be used for deploying the quickstart in an easy way. I'm not convinced about saying it's a `core custom resource for Entando infrastructure` . You can deploy Entando 6 even without EntandoCompositeApp.
Other custom resources are required (and therefore "core" of the infrastructure) to deploy Entando 6, like the EntandoKeycloakServer, or the EntandoApp.
What do you think?

File: vuepress/docs/tutorials/devops/default-database.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+1. open the chosen file and search for `EntandoCompositeApp` custom resource
+2. identify the component to update in the related list
+3. update the `spec.dbms` property with the desired value (you can specify different DBMS for different components)
+
+Repeat previous steps for all components you need to change used DBMS on.
+
+Valid values for `spec.dbms` property are: `none`, `postgresql`, `mysql`, `oracle`.
+
+`none` value will result in using an embedded database with in-file persistence strategy.
+If `none` (embedded database) is choosen each interested container requires a [PVC](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) mounted on path `/opt/jboss/keycloak/standalone/data/`
```
Comment (Kerruba) [inline vuepress/docs/tutorials/devops/default-database.md:26]: mounted path for the PVC will be different between container. The reported one is valid only for keycloak

Comment (nshaw) [reply vuepress/docs/tutorials/devops/default-database.md:26]: Maybe just drop the "mounted on path..." part?

Comment (Kerruba) [inline vuepress/docs/tutorials/devops/default-database.md:26]:
```suggestion
```
If you agree, I would remove this info as really a detail. You can't customize the PVC anyway from a user point of view.

**Pull request merged**